chrome/content/zotero/xpcom/data/feed.js

```diff
@@ -89,10 +112,22 @@ Zotero.defineProperty(Zotero.Feed.prototype, 'isFeed', {
 Zotero.defineProperty(Zotero.Feed.prototype, 'libraryTypes', {
 	value: Object.freeze(Zotero.Feed._super.prototype.libraryTypes.concat(['feed']))
 });
+Zotero.defineProperty(Zotero.Feed.prototype, 'unreadCount', {
+	get: function() this._feedUnreadCount
+});
+Zotero.defineProperty(Zotero.Feed.prototype, 'updating', {
+	get: function() this._updating,
+	set: function(v) {
+		if (!v == !this._updating) return; // Unchanged
+		this._updating = !!v;
+		Zotero.Notifier.trigger('statusChanged', 'feed', this.id);
+	}
+});
 
 (function() {
 // Create accessors
-let accessors = ['name', 'url', 'refreshInterval', 'cleanupAfter'];
+let accessors = ['name', 'url', 'refreshInterval', 'cleanupAfter', 
+	'lastCheck', 'lastUpdate', 'lastCheckError'];
```

Comment:
Yes, however `_set()` has some handling code for lastCheckError and generally there has been code like `this.lastCheck = Zotero.Date.dateToSQL(new Date(), true);` within the class, which looked valid but did nothing. `_set()` could be used within the class, but I went over other classes within Zotero and did not find any variables without setters, so decided this is the more consistent (if less encapsulating) solution.